src/modules/core/components/Fields/Select/Select.css

```diff
@@ -100,20 +101,32 @@
 
 .themeGrey {
   composes: baseSelect;
-  padding: 10.5px 5px 10.5px 15px;
+  padding: 10.5px 15px 10.5px 15px;
```

Comment:
I know you didn't add this, but we really shouldn't use fractional pixel values.

I don't even know how this is supposed to work. A pixel is a pixel.

Reply:
Issue raised